src/shogun/lib/parameter_observers/ParameterObserverCV.cpp

```diff
@@ -164,7 +164,7 @@ CrossValidationStorage* CParameterObserverCV::get_observation(int run) const
 	return obs;
 }
 
-const size_t CParameterObserverCV::get_num_observations() const
+const int32_t CParameterObserverCV::get_num_observations() const
```

Comment:
Need some checks here as sizet is unsigned

Reply:
What kind of checks? This method should just return the total number of observations, or am I missing something? :/

Reply:
`size_t` is unsigned so it can represent numbers that cannot be converted to `int32_t`. That is minor as there likely won't be that many observations. But the compiler will generate warnings about such implicit conversions

Reply:
Okay, so, since `std::vector::size` returns a `size_t` value, I will keep also the return type of the function equal to `size_t`.